packages/components-native/src/__mocks__/__mocks.ts

```diff
@@ -44,3 +48,15 @@ jest.mock("react-intl", () => {
     useIntl: () => intl,
   };
 });
+
+jest.mock("../hooks/useIsScreenReaderEnabled", () => ({
+  useIsScreenReaderEnabled: () => false,
+}));
+
+// By default React Native mocks the return value of `addEventListener` using
+// jest.fn(). See https://github.com/facebook/react-native/blob/0.69-stable/jest/setup.js#L124
+// This provides an implementation of .remove for useEffect's that do sbuscribe and
```

Comment:
Just noticed this as I was perusing your PR: :)
```suggestion
// This provides an implementation of .remove for useEffect's that do subscribe and
```